filament/backend/src/vulkan/VulkanDriver.cpp

```diff
@@ -874,6 +874,8 @@ bool VulkanDriver::isWorkaroundNeeded(Workaround workaround) {
             return deviceProperties.vendorID == 0x5143; // Qualcomm
         case Workaround::ALLOW_READ_ONLY_ANCILLARY_FEEDBACK_LOOP:
             return true;
+        case Workaround::ADRENO_UNIFORM_ARRAY_CRASH:
+            return false;
```

Comment:
We need this in the Metal backend as well.

Reply:
Hello, pixelflinger, I think I've added the same code in MetalDriver.mm.